File: app/src/main/java/com/drivingschoolrwandaapp/models/request/IremboSpecialRequest.java
package com.drivingschoolrwandaapp.models.request;

public class IremboSpecialRequest {
    private String serviceName;
    private String category;
    private String applicantName;
    private String applicantPhone;
    private String nationalId;
    private String description;

    public IremboSpecialRequest(String serviceName, String category, String applicantName, String applicantPhone, String nationalId, String description) {
        this.serviceName = serviceName;
        this.category = category;
        this.applicantName = applicantName;
        this.applicantPhone = applicantPhone;
        this.nationalId = nationalId;
        this.description = description;
    }

    public String getServiceName() {
        return serviceName;
    }

    public void setServiceName(String serviceName) {
        this.serviceName = serviceName;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getApplicantName() {
        return applicantName;
    }

    public void setApplicantName(String applicantName) {
        this.applicantName = applicantName;
    }

    public String getApplicantPhone() {
        return applicantPhone;
    }

    public void setApplicantPhone(String applicantPhone) {
        this.applicantPhone = applicantPhone;
    }

    public String getNationalId() {
        return nationalId;
    }

    public void setNationalId(String nationalId) {
        this.nationalId = nationalId;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/request/IremboSpecialRequest.kt
package com.drivingschoolrwandaapp.models.request

import com.google.gson.annotations.SerializedName

data class IremboSpecialRequest(
    @SerializedName("serviceName") var serviceName: String = "",
    @SerializedName("category") var category: String = "",
    @SerializedName("applicantName") var applicantName: String = "",
    @SerializedName("applicantPhone") var applicantPhone: String = "",
    @SerializedName("nationalId") var nationalId: String = "",
    @SerializedName("description") var description: String = ""
)
